diff --git a/node_modules/source-map/lib/read-wasm.js b/node_modules/source-map/lib/read-wasm.js
index e81c177..9b250ba 100644
--- a/node_modules/source-map/lib/read-wasm.js
+++ b/node_modules/source-map/lib/read-wasm.js
@@ -1,7 +1,7 @@
 /* Determine browser vs node environment by testing the default top level context. Solution courtesy of: https://stackoverflow.com/questions/17575790/environment-detection-node-js-or-browser */
 const isBrowserEnvironment = (function() {
     // eslint-disable-next-line no-undef
-    return (typeof window !== "undefined") && (this === window);
+    return typeof window !== "undefined" && typeof window.document !== "undefined"
 }).call();
 
 if (isBrowserEnvironment) {
